CMakeLists.txt

```diff
@@ -231,12 +231,6 @@ if(MSVC)
   add_definitions(-D_CRT_SECURE_NO_WARNINGS)
 endif()
 
-if("${JPEGXL_ENABLE_FUZZERS}" OR "${JPEGXL_ENABLE_COVERAGE}")
```

Comment:
Was this removal on purpose?

Reply:
Yes, a quick `git grep` suggests that we don’t actually use it anywhere, and it is doubtful that we should.